src/services/event.ts

```diff
@@ -444,3 +445,112 @@ export async function findAllEvents(): Promise<EventShape[]> {
     raw: true,
   });
 }
+
+const splitPipe = (str: string) => str.split('\n').map((s) => s.trim()).filter(Boolean);
+
+const mappings: Record<string, string> = {
+  Audience: 'audience',
+  Creator: 'creator',
+  'Edit Title': 'eventName',
+  'Event Duration/#NC Days of Support': 'eventDuration',
+  'Event ID': 'eventId',
+  'Overall Vision/Goal for the PD Event': 'vision',
+  'Reason for Activity': 'reasons',
+  'Target Population(s)': 'targetPopulations',
+  'Event Organizer - Type of Event': 'eventOrganizer',
+};
+
+const toSplit = ['targetPopulations', 'reasons'];
```

Comment:
Do we assert the new array created in a test?

Reply:
Again this just means to validate the resulting array in a unit test.

Reply:
The tests now check for this